static/js/sidebar.js

```diff
@@ -95,11 +95,19 @@ var SidebarView = Backbone.View.extend({
                     continue;
                 }
 
-                var entryHtml = this.sidebarItemTemplate({
-                    created: moment(markerModel.get("created")).format("LLLL"),
-                    type: SUBTYPE_STRING[markerModel.get("subtype")],
-                    icon: markerView.getIcon()
-                });
+                if (isRetina){
```

Comment:
Factor out the common part please, don't repeat it. It's better to just assign to a local variable `icon` and then create the entry with it.